master/buildbot/config.py

```diff
@@ -449,7 +451,7 @@ def load_caches(self, filename, config_dict):
             if not isinstance(caches, dict):
                 error("c['caches'] must be a dictionary")
             else:
-                valPairs = caches.items()
+                valPairs = list(iteritems(caches))
```

Comment:
This is very true!. I'll change it into just a simple ´´´for k,.v in iteritems(caches)``` and remove that line